concepts/components-events-groups.md

```diff
@@ -40,9 +40,9 @@ Here is an example:
 }
 ```
 
-All component groups are custom-created. You cannot use component groups from other entities in your own entity. In the example above, `minecraft:cat_persian` is *not* a component. You cannot use it in your entity. If you want the `cat_persian` functionality in your entity, you must create your own group, or add the components directly into the your entities `components` object.
+All component groups are custom-created. You cannot use component groups from other entities in your entity. In the example above, `minecraft:cat_persian` is *not* a component. You cannot use it in your entity. If you want the `cat_persian` functionality in your entity, you must create your group, or add the components directly into your entities  `components` object.
```

Comment:
`Own` was also correct here. For sure in the second case, and arguably in the first case. Its important to stress "own", to differentiate between ANY group, and one you created.